DependencyInjection/SpClient.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Italia\SpidSymfonyBundle\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\ContainerInterface as Container;
+use Italia\Spid\Sp;
+
+class SpClient
+{
+
+    private $spClientConfigured;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function __construct(Container $container)
+    {
+
+        $configuration = [
```

Comment:
Not 100% sure, but `$container->getParameter` doesn't cache
Assign `$container->getParameter('spid')` and then retrieve the values from the hashmap you get